package-lock.json

```diff
@@ -2242,8 +2242,7 @@
         "ansi-regex": {
```

Comment:
@sachinma package-lock.json is being auto modified. Do you think we should include this because sometimes it may create issues due to wrong installs by npm

Reply:
The changes look simple. We can keep it